src/backend/commands/createas.c

```diff
@@ -1272,6 +1306,9 @@ check_ivm_restriction_walker(Node *node, check_ivm_restriction_context *ctx, int
 					ereport(ERROR,
 							(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 							 errmsg("system column is not supported on incrementally maintainable materialized view")));
+				/* if subquery reffer vars of  parent tables */
+				if (variable->varlevelsup > 0 && ctx->has_exists_subquery)
+					ctx->exists_quals = lappend(ctx->exists_quals, node);
```

Comment:
It is better to use exists_qual_vars instared of exists_quals?

Reply:
I would like to fix 'exists_qual_vars' instead of 'exists_quals'.